### ISSUE TITLE
Accessibility: long-press to expand FABs

### ISSUE THREAD

**Issue by shortwavesurfer2009** [thunder-app/thunder#527]
The FABs should be expandable with a long press as swipes and screen readers such as talkback do not work well together. When the FAB expands the focus for screen readers should be placed on the top most item, followed by the next down, and so on. The very bottom option needs to be "close" or "collapse"
![Screenshot_20230802-103720_Thunder](https://github.com/thunder-app/thunder/assets/116814522/9fadd2d9-ae78-4f18-86d0-e56495635e05)

**Comment by hjiangsu** [comment]
I think the original plan here was to allow for a long press to trigger a secondary action in the future. I think we can maybe accommodate both use-cases by doing this:

Introduce a new setting: Long press action on FAB

The available options for these new settings would be:
- Expand/collapse FAB which would fulfill this accessibility issue
- Trigger secondary action (in which case, you could choose your secondary action from a list of actions)

**Comment by shortwavesurfer2009** [comment]
> I think the original plan here was to allow for a long press to trigger a secondary action in the future. I think we can maybe accommodate both use-cases by doing this:
> 
> Introduce a new setting: Long press action on FAB
> 
> The available options for these new settings would be:
> 
>     * Expand/collapse FAB which would fulfill this accessibility issue
> 
>     * Trigger secondary action (in which case, you could choose your secondary action from a list of actions)

Seems fine to me. What would the default be set to?

**Comment by CTalvio** [comment]
Default action could be either nothing, or expand. The long press won't have anything communicating what it does to most users, so should only trigger a secondary action for users who manually set one, and hence know what it will do.

**Comment by shortwavesurfer2009** [comment]
> Default action could be either nothing, or expand. The long press won't have anything communicating what it does to most users, so should only trigger a secondary action for users who manually set one, and hence know what it will do.

Personally i think it should be set to expand by default as this would best help for accessibility.
